Cargo.toml

```diff
@@ -56,9 +54,10 @@ wit-parser = "0.13.1"
 testresult = "0.3.0"
 
 [features]
-default = []
+default = ["cli-interactive"]
 postgres = ["warg-server/postgres"]
-not-cli = ["warg-client/not-cli"]
+cli-interactive = ["warg-client/cli-interactive"]
+not-interactive-cli = []
```

Comment:
Unused?

Reply:
Is there a good way to specify that you don't want the `cli-interactive` feature if there isn't an empty set?

Reply:
Fixed